pkg/models/device.go

```diff
@@ -5,7 +5,21 @@ package models
 type Device struct {
 	ID            string `json:"id"`
 	VersionID     string `json:"versionId"`
+	Version       string `json:"version"`
+	AppID         string `json:"appId"`
 	DeviceID      string `json:"deviceId"`
 	DeviceVersion string `json:"deviceVersion"`
 	DeviceType    string `json:"deviceType"`
+	AppName       string `json:"appName,omitempty"`
+}
```

Comment:
Since we will create new Apps now, then is required the name. 
Also, I think that other attributes here should be `omitempty` too as `DeviceVersion` and `DeviceType `